### PR TITLE
TF-Lite | Add custom objects when loading keras model

### PR DESCRIPTION
**Pull request by julianzq**
If the exported tf.keras model has any tf operations or Lambda layers that use tf internally loading the model will fail with the error `tf is not defined`.
To fix that it is important to load tf as a custom object when loading a keras model like that - `cusotm_objects={'tf': tf)`.

### CODE REVIEW

**Comment by israelg99** [comment]
I signed it!

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @gargn: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @gargn: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @gargn: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @gargn: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @gargn: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/lite/python/lite.py [hunk 1]
```diff
@@ -348,7 +348,8 @@ def from_keras_model_file(cls,
                             model_file,
                             input_arrays=None,
                             input_shapes=None,
-                            output_arrays=None):
+                            output_arrays=None,
+                            custom_objects=None):
```
Comment (geek101) [inline tensorflow/lite/python/lite.py]: Instead of adding `custom_objects` is it possible to add `**kwargs` to `from_keras_model_file()`?. See below for rest of the comments.

File: tensorflow/lite/python/lite.py [hunk 2]
```diff
@@ -367,7 +368,7 @@ def from_keras_model_file(cls,
     """
     _keras.backend.clear_session()
     _keras.backend.set_learning_phase(False)
-    keras_model = _keras.models.load_model(model_file)
+    keras_model = _keras.models.load_model(model_file, custom_objects=custom_objects)
```
Comment (geek101) [inline tensorflow/lite/python/lite.py]: And let this call be `_keras.models.load_model(model_file, **kwargs)`.
This is a way to make the API `from_keras_model_file()` somewhat future proof w.r.t underlying Keras `load_model()` API change?.

**Comment by Hazarapet** [comment]
Any news from this?

**Comment by gargn** [comment]
This functionality has been added by https://github.com/tensorflow/tensorflow/commit/09deaeb03ca4ceb40cf600a337083e2054e65390.
